.github/workflows/deploy-to-chromatic.yml

```diff
@@ -34,3 +31,51 @@ jobs:
           exitZeroOnChanges: true
           exitOnceUploaded: true
           autoAcceptChanges: 'main'
+  pr-deploy-to-chromatic:
+    if: github.event.issue.pull_request && contains(github.event.comment.body, '/chromatic')
```

Comment:
Is this the only time a chromatic build would be deployed (if the comment contains this string)?

Reply:
Yep, you would have to comment "/chromatic"

Reply:
Opted for a different approach to continue to get the native GitHub status checks